tracker/partidachess.py

```diff
@@ -0,0 +1,52 @@
+from tablero import *
+
+class PartidaChess:
+   '''
+   Clase encargada de almacenar los datos de una partida de ajedrez
+        
+   Atributos
+   ---------
+        
+    _nombres_blancas : str
+    	Nombre de usuario de jugador en blancas
+    _nombre_negras : str
+    	Nombre de usuario de jugador en negras
+    _ganador : str
+        Nombre del ganador de la partida
+    _movimientos_partida : list[tablero]
+    	Lista de tableros (movimientos) de los jugadores durante la partida
+   '''
+
+   def __init__(self,nombre_blancas,nombre_negras,ganador,movimientos):
+       
+        '''
+    	Construye un objeto de tipo PartidaChess
+    	
+    	Atributos
```

Comment:
Esto no serían atributos sino parámetros. De todas formas yo lo escribiría en inglés, que es como está en la [guía de estilo de numpy](https://numpydoc.readthedocs.io/en/latest/format.html), el formato que parece que utilizas.
```suggestion
    	Parámetros
```